dependency-check-maven-config.xml

```diff
@@ -542,4 +542,12 @@
         </notes>
         <cve>CVE-2016-8739</cve>
     </suppress>
-</suppressions>
\ No newline at end of file
+    <!-- Remove this suppression when DDF-3440 is worked -->
```

Comment:
Approving, but we should include a link to this block in the Jira ticket so it is not lost.

Reply:
done